test/driver-e2e-specs.js

```diff
@@ -17,9 +17,11 @@ const caps = {platformName: "Fake", deviceName: "Fake", app: TEST_FAKE_APP};
 
 describe('FakeDriver - via HTTP', () => {
   let server = null;
+  let baseUrl;
   before(async () => {
     if (shouldStartServer) {
       let args = {port: TEST_PORT, host: TEST_HOST};
+      baseUrl = `http://${TEST_HOST}:${TEST_PORT}/wd/hub/session`;
```

Comment:
If we use a running server this means that `baseUrl` will remain undefined. On line 20 it should just be 
```
const baseUrl = `http://${TEST_HOST}:${TEST_PORT}/wd/hub/session`;
```